src/main/java/edu/alysonhudak/advancedjava/stockservice/YahooStockService.java

```diff
@@ -0,0 +1,87 @@
+package edu.alysonhudak.advancedjava.stockservice;
+
+import yahoofinance.Stock;
+import yahoofinance.YahooFinance;
+import yahoofinance.histquotes.HistoricalQuote;
+import yahoofinance.histquotes.Interval;
+
+import javax.validation.constraints.NotNull;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.List;
+
+/**
+ * This class models a stock service that gets stock data from the Yahoo Finance web service
+ * (https://financequotes-api.com/).  It does not implement interface {@code StockService}. Use the adapter
+ * class YahooStockServiceAdapter if need access to its functionality from a class that does.
+ *
+ * @author Alyson Hudak
+ */
+public class YahooStockService  {
```

Comment:
So this class ends with StockService so by convention it should implement the StockService.
Moreover, The StockService class should internally use the YahooStockServiceAdapter class or it's functionality here.  Let me know if that makes sense. because it is an important part of the Adapter pattern.